fedora/client/openidbaseclient.py

```diff
@@ -154,6 +160,20 @@ def __init__(self, base_url, login_url=None, useragent=None, debug=False,
 
         # python-requests session.  Holds onto cookies
         self._session = requests.session()
+
+        # Also hold on to retry logic.
+        # http://www.coglib.com/~icordasc/blog/2014/12/retries-in-requests.html
+        if retries is not None:
+            prefixes = ['http://', 'https://']
+            for prefix in prefixes:
+                self._session.mount(prefix, requests.adapters.HTTPAdapter(
+                    max_retries=Retry(
+                        total=retries,
+                        status_forcelist=[500],
```

Comment:
Yeah, it looks like you have to explicitly list them.  I'll add all the 500 codes I can find.